rotkehlchen/api/v1/schemas.py

```diff
@@ -945,12 +946,37 @@ def make_history_base_entry(
         ) -> dict[str, Any]:
             return {'event': EthWithdrawalEvent(**data)}
 
+    class CreateAssetMovementEventSchema(BaseSchema):
+        event_type = SerializableEnumField(
+            enum_class=HistoryEventType,
+            exclude_types=[
+                event_type for event_type in HistoryEventType
+                if event_type not in (HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL)
+            ],
+            required=True,
+        )
+        asset = AssetField(required=True, expected_type=Asset, form_with_incomplete_data=True)
+        event_identifier = fields.String(required=False, load_default=None)
+        unique_id = fields.String(required=False, load_default=None)
+        extra_data = fields.Dict(load_default=None)
+        location = LocationField(required=True)
+        is_fee = fields.Boolean(required=True)
```

Comment:
again since this is from the user it makes no sense for them to repeat same things for fee. Instead add an optional fee amount (andif asset can be different, optional fee asset) and we can turn it into the second event in the backend.